deploy/catalog/templates/catalog.yaml

```diff
@@ -15,6 +15,24 @@ spec:
         imagePullPolicy: Always
         ports:
         - containerPort: 10000
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 10000
+          failureThreshold: 3
+          initialDelaySeconds: 5
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 2
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 10000
+          failureThreshold: 1
```

Comment:
@martinmaly I know you weren't pushing for any particular change, but your question brought me to the realization that readiness was irrelevant here, so I acted on that new insight. Thanks for the review!